### ISSUE TITLE
NHC hangs when ps has long commands in output

### ISSUE THREAD

**Issue by treydock** [mej/nhc#52]
This is an extreme edge case.  We have a user who seems to dump a perl script as the command for their job.  So /proc/$PID/cmdline ends up being an entire perl script:

```
# wc /proc/30638/cmdline 
3754 6542 64551 /proc/30638/cmdline
```

This seems to cause problems for NHC when using check_ps_service.  Running the actual ps command does not hang or take a long time.

```
[root@n0410 ~]# nhc -l- -v
<SNIP>
Running check:  "check_ps_service -r -d sssd_be sssd"
NHC watchdog timer 23612 (30 secs) has expired.  Signaling NHC:  kill -s ALRM -- -23611
ERROR:  nhc:  Health check failed:  Script timed out while executing "check_ps_service -r -d sssd_be sssd".
NHC watchdog timer 23612 terminating NHC:  kill -s TERM -- -23611
[root@n0410 ~]# NHC watchdog timer 23612 terminating NHC with extreme prejudice:  kill -s KILL -- -23611
```

I've saved the output of the NHC ps command, let me know if it would be useful to send directly via email as a reproducer.

**Comment by mej** [comment]
In `nhc_ps_gather_data()` [here](https://github.com/mej/nhc/blob/master/scripts/lbnl_ps.nhc#L62), the `ps` command NHC uses is defined.  Can you please try adding `--width 1024` between the `ps` and the `axo`?  That should set the maximum line length to 1k, which I think should be reasonable.  See if that speeds things up.  If so, feel free to submit a PR if you'd like. :-)

**Comment by treydock** [comment]
@mej Finally got around to testing this and using `--width 1024` solved the hangs.  Will open PR.

**Comment by mej** [comment]
Fixed in #64 by @treydock and merged as mej/nhc@4d3950e.
